sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -289,4 +341,22 @@ public void setRequest(final @Nullable Request request) {
   Span getRoot() {
     return root;
   }
+
+  private static final class FinishStatus {
+    private final boolean isFinishing;
+    private final @Nullable SpanStatus spanStatus;
+
+    static @NotNull FinishStatus notFinished() {
+      return new FinishStatus(false, null);
```

Comment:
Looking at the usage of this method we'll need a instance of this for each tracer. On Spring we'll have on per request so could declare this as a field and just returned that single instance?

Reply:
Done! Thanks good point